packages/pocket-ic/src/nonblocking.rs

```diff
@@ -78,6 +78,8 @@ pub struct PocketIc {
     http_gateway: Option<HttpGatewayInfo>,
     server_url: Url,
     reqwest_client: reqwest::Client,
+    // do not delete the instance when dropping this handle if this handle was created for an existing instance
+    keep: bool,
```

Comment:
```suggestion
    is_handle: bool,
```
Keep can be ambiguous

Reply:
To me, "is_handle" is more confusing since the `PocketIc` struct that created the instance is also a handle. Maybe "owns_instance"?

Reply:
FWIW, I understand `is_handle` worse than `keep`.